x-pack/metricbeat/module/googlecloud/README.md

```diff
@@ -0,0 +1,45 @@
+# Main concepts
+Stackdriver == Cloudwatch :)
+
+* Light modules
+* Stackdriver requests are based on *gimme last 5s or 5m of compute.cpu.usage* so lots of requests are needed.
+* *Some* metadata is included on Stackdriver, not all.
+* If you want the metadata that is missing, some interfaces must be implemented.
+* Stackdriver returns an array of points, each with a timestamp, values and labels. Their only correlation is the metric. So you can get 5 points, of 5 different minutes in time, 2 of them with labels a=b and 3 with c=d, but all of them will be `cpu.usage.pct` for example.
+* There are 4 possible types of metadata available: User, Label, System and Metadata. This is called **Metadata** in code.
+* In code there's the concept of Event ID. It's a "hash" of the timestamp and the Metadata. This is an "algorithm" which tries to group metric points into the same events. The idea is that if we get `X points *
```

Comment:
Just want to make it clearer 😬 Does the metadata here used for creating event ID includes User, Label and System metadata?

Reply:
I asked this question live to @sayden and the answer is "yes" :)